src/views/Stake/Stake.jsx

```diff
@@ -67,19 +67,19 @@ function Stake() {
     return state.app.fiveDayRate;
   });
   const ohmBalance = useSelector(state => {
-    return state.app.balances && state.app.balances.ohm;
+    return state.account.balances && state.account.balances.ohm;
   });
   const oldSohmBalance = useSelector(state => {
-    return state.app.balances && state.app.balances.oldsohm;
+    return state.account.balances && state.account.balances.oldsohm;
   });
   const sohmBalance = useSelector(state => {
-    return state.app.balances && state.app.balances.sohm;
+    return state.account.balances && state.account.balances.sohm;
   });
   const stakeAllowance = useSelector(state => {
```

Comment:
No Cleaner way to do this eh?

Reply:
I was just replicating functionality in this PR.  Im sure there's a cleaner way to do it